packages/ember-test-app/app/templates/promo.hbs

```diff
@@ -1,3 +1,115 @@
-<div class="container p-3">
+{{! <div class="container p-3">
   <Mktg::Promo />
-</div>
\ No newline at end of file
+</div> }}
+<main data-theme="marketing" class="bg-primary-subtle vw-100">
```

Comment:
Not 100% sure why, but I get a horizontal scroll bar when running locally. It looks like this is too wide?

Reply:
Yeah I can remove the vw-100, I was doing that for local developement but forgot to remove it!